.github/workflows/js.yml

```diff
@@ -111,7 +104,7 @@ jobs:
       - name: Checkout Arrow
         uses: actions/checkout@v1
```

Comment:
`v1` -> `v2`

Reply:
Thanks for catching this, I've created https://issues.apache.org/jira/browse/ARROW-15140 and I'll follow up with that shortly.

Reply:
Oh, ha it was _just that one_ too! Anyway: https://github.com/apache/arrow/pull/11983/files